WeakAurasOptions/CommonOptions.lua

```diff
@@ -1123,9 +1131,38 @@ local function PositionOptions(id, data, _, hideWidthHeight, disableSelfPoint, g
       order = 72,
       image = function() return "", 0, 0 end,
       hidden = function()
-        return IsParentDynamicGroup() or not (data.anchorFrameType == "SCREEN" or data.anchorFrameType == "UIPARENT" or data.anchorFrameType == "MOUSE" or IsGroupByFrame())
+        return
+          IsParentDynamicGroup() or
+          not (
+            data.anchorFrameType == "SCREEN" or
+            data.anchorFrameType == "UIPARENT" or
+            data.anchorFrameType == "MOUSE" or
+            data.anchorFrameType == "ACTION_BUTTON_SPELL" or
+            IsGroupByFrame()
+          )
       end,
     },
+    anchorFrameActionButtonSpell = {
+      type = "input",
+      width = WeakAuras.doubleWidth,
+      name = L["Spell"],
+      order = 72,
+      set = function(info, value)
+        if (WeakAuras.IsSpellKnownIncludingPet(value)) then
+          data.anchorFrameActionButtonSpellId = select(7, GetSpellInfo(value))
```

Comment:
due to some magic & unwise code elsewhere, if a user imports something which uses this feature, & then later updates it & that update changes this value, it will appear as a change to the display tab. It should probably be added to the anchor category here https://github.com/WeakAuras/WeakAuras2/blob/95ef628bd99de1a49675f70df9353fabf8db83e4/WeakAuras/Types.lua#L3381